share/ci/generate_reduced_matrix.sh

```diff
@@ -66,6 +66,13 @@ elif [ "$PIC_INPUTS" == "pmacc_header" ] ; then
     # create header validation test for PMacc
     echo "pmacc_header" | tr " " "\n" | n_wise_generator.py $@ --limit_boost_version $ADDITIONAL_GENRATOR_FLAGS
   fi
+elif [ "$PIC_INPUTS" == "picongpu_header" ] ; then
+  if [ $commit_pmacc_only -ne 1 ] ; then
+    # create unit tests for PIConGPU
+    echo "picongpu_header" | tr " " "\n" | n_wise_generator.py $@ --limit_boost_version $ADDITIONAL_GENRATOR_FLAGS
```

Comment:
```suggestion
    echo "picongpu_header" | tr " " "\n" | n_wise_generator.py $@ --limit_boost_version $ADDITIONAL_GENERATOR_FLAGS
```
There's a typo in that variable's name (in general, not just this line). No need to fix immediately, but one could...

Reply:
I will do this change in a separate PR because #5108 is fixing a bug and both PRs will collide if I rename the variable. After both are merged I can perform the renaming.